src/assets/stylesheets/components/site/_educator-mode-toggle.scss

```diff
@@ -0,0 +1,25 @@
+.landing-page-toggle {
+  max-width: 20rem;
+}
+
+.educator-toggle-container {
+  display: flex;
+
+  .md-switch-container {
+    flex-shrink: 0;
+  }
+}
+
+.educator-passphrase-input {
+  width: 0%;
+  margin-left: 0;
+  transition-delay: 0.25s;
+  transition-timing-function: cubic-bezier(0.39, 0.575, 0.565, 1);
+  transition-duration: 0.5s;
+  transition-property: width, margin-left;
```

Comment:
The linter is complaining about not using the `transition` shorthand here, but I feel like in a lot of cases, this one included, the property by property listing of transition stuff is clearer and more terse, so I'm editing the linting rule to no longer apply to transition properties.